internal/meta/req.go

```diff
@@ -35,6 +37,9 @@ func FromTransportRequest(ctx context.Context, req *transport.Request) yarpc.Req
 // ToTransportRequest fills the given transport request with information from
 // the given ReqMeta. The Context associated with the ReqMeta is returned.
 func ToTransportRequest(reqMeta yarpc.CallReqMeta, req *transport.Request) context.Context {
+	if reqMeta == nil {
+		log.Panic("reqMeta must not be nil")
```

Comment:
Yeah. Doesn't it make sense for the app to die if you didn't provide a `ReqMeta`? The ReqMeta contains the context and in case of non-Thrift, the name of the procedure to be called. We can't really make a request without that information.